runner/Dockerfile

```diff
@@ -47,7 +47,7 @@ RUN apt update -y \
 # arch command on OS X reports "i386" for Intel CPUs regardless of bitness
 RUN export ARCH=$(echo ${TARGETPLATFORM} | cut -d / -f2) \
     && if [ "$ARCH" = "arm64" ]; then export ARCH=aarch64 ; fi \
-    && if [ "$ARCH" = "i386" ]; then export ARCH=x86_64 ; fi \
+    && if [ "$ARCH" = "amd64" ] || [ "$ARCH" = "i386" ]; then export ARCH=x86_64 ; fi \
```

Comment:
We probably need to do this over [here](https://github.com/actions-runner-controller/actions-runner-controller/blob/a9421edd465b327ed6379209087f88119649189e/runner/Dockerfile.dindrunner#L111) too, right?

Reply:
Yes, you're right. 
I'm going to fix it, too :)

Reply:
I've fixed it!
cba8691bebb0b891fa11f1c4b61fdbb101e0b261